src/main/java/net/minecraftforge/event/world/AddBuiltinRegistryEvent.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) Forge Development LLC and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.event.world;
+
+import com.mojang.serialization.Codec;
+import net.minecraft.core.Registry;
+import net.minecraft.core.RegistryAccess;
+import net.minecraft.resources.ResourceKey;
+import net.minecraftforge.eventbus.api.Event;
+import net.minecraftforge.fml.event.IModBusEvent;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.function.BiConsumer;
+
+/**
+ * <p>
+ *    This event is fired on the Mod event bus at the time of FMLLoadCompleteEvent
+ * </p>
+ * <p>
+ *    This event is used to register the Codec to a data-pack-registry. All Registries passed here should be marked as data-pack-registries
+ *    with RegistryBuilder#dataPackRegistry.
+ * </p>
+ * <p>
+ *    You should avoid the usage of {link AddBuiltinRegistryEvent#addRegistry(ResourceKey, Codec, Codec)},
+ *    because clients missing the required network codec can cause strange issues
+ * </p>
+ */
+public class AddBuiltinRegistryEvent extends Event implements IModBusEvent
```

Comment:
Use `{@link}` more here, instead of just dropping capitalized names.
Missing a period on the last sentence.
Last sentence is also not specific. What strange issues? When will a client being missing the network codec?